extension/src/openvic-extension/singletons/GitInfo.cpp

```diff
@@ -0,0 +1,80 @@
+
+#include "GitInfo.hpp"
+
+#include <godot_cpp/core/error_macros.hpp>
+
+#include "openvic-extension/utility/ClassBindings.hpp"
+
+#include "gen/git_info.gen.hpp"
+#include "godot_cpp/classes/global_constants.hpp"
+
+using namespace godot;
+using namespace OpenVic;
+
+GitInfo::GitInfo() {
+	ERR_FAIL_COND(_singleton != nullptr);
+	_singleton = this;
+
+	_game_commit_hash = GAME_COMMIT_HASH.data();
+	_game_commit_short_hash = _game_commit_hash.substr(0, 7);
+	_game_tag = GAME_TAG.data();
+	_game_release = GAME_RELEASE.data();
```

Comment:
```suggestion
	_game_commit_hash = Utilities::std_to_godot_string(GAME_COMMIT_HASH);
	_game_commit_short_hash = _game_commit_hash.substr(0, 7);
	_game_tag = Utilities::std_to_godot_string(GAME_TAG);
	_game_release = Utilities::std_to_godot_string(GAME_RELEASE);
```
So here we initialise `StringName`s from `char const*`s obtained via `std::string_view`s. We know in practice that these C-strings will always be null-terminated (as the `std::string_view`s are intialised from string literals), but would the `std::string_view` actually include the null character in its length?
To be absolutely sure we avoid any such problem in the future, we could use `Utilities::std_to_godot_string` in all 3 cases here rather than just getting `data()` (and add `#include "openvic-extension/utility/Utilities.hpp"` above).